ctrd/snapshot.go

```diff
@@ -13,6 +13,15 @@ import (
 
 const (
 	defaultSnapshotterName = "overlayfs"
+
+	// TypeLabelKey is the key of label type=image in Snapshotter metadata Info.Labels, with which indicates the snapshot for image.
+	TypeLabelKey = "type"
+
+	// ImageType is the label value
+	ImageType = "image"
+
+	// SnapshotLabelContextKey is the key which is stored in context to transfer to containerd snapshotter
+	SnapshotLabelContextKey = "containerd.io/snapshot/labels"
```

Comment:
Can this part move to ctrd/snapshotter_key.go

Reply:
Move it!